src/Backoffice/Employees/Domain/Actions/StoreEmployeeAction.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employees\Domain\Actions;
+
+use Lightit\Backoffice\Employees\App\Requests\StoreEmployeeRequest;
+use Lightit\Backoffice\Employees\Domain\Models\Employee;
+
+class StoreEmployeeAction
+{
+    public function execute(StoreEmployeeRequest $request): Employee
```

Comment:
```suggestion
    public function execute(StoreEmployeeRequest $request): Employee
```
It's not a good idea to rely on the Form Request, because now it becomes quite uncomfortable to be able to use this action anywhere else where I don't have an instance of `StoreEmployeeRequest`. It'd be a better approach to pass a DTO so that the action can be used anywhere in the app by simply creating an instance of a DTO, which is way simpler than creating an instance of `StoreEmployeeRequest` from scratch.